client/my-sites/google-my-business/stats/index.js

```diff
@@ -0,0 +1,159 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize, translate } from 'i18n-calypso';
+import { random, range } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import Chart from 'components/chart';
+import DocumentHead from 'components/data/document-head';
+import Location from '../select-location/location';
+import LocationType from '../select-location/location-type';
```

Comment:
We may want to import `Location` as `GoogleMyBusinessLocation` similarly to what we do in the `GoogleMyBusinessSelectLocation` component. Since the former is now used in two different pages we could also move it in its own folder.